raiden/settings.py

```diff
@@ -44,3 +44,6 @@
 
 ORACLE_BLOCKNUMBER_DRIFT_TOLERANCE = 3
 ETHERSCAN_API = 'https://{network}.etherscan.io/api?module=proxy&action={action}'
+
+DEFAULT_LIMITS_CONTRACT_VERSION = '0.4.0'
```

Comment:
Why call them default here? Wouldn't it make more sense to name these two as:

```python
RED_EYES_CONTRACT_VERSION = '0.4.0'
DEVELOPMENT_CONTRACT_VERSION = '0.3._'
```

Notice that `0.3._` also has limits.

Reply:
Good point. Fixed.